Assets/Treasure/TDK/Runtime/Utils/TreasureLauncherUtils.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Treasure
+{
+    public class TreasureLauncherUtils
+    {
+        private static bool hasParsedCommandLineArgs = false;
+        
+        private static string tdkAuthToken = null;
+
+        public static string GetLauncherAuthToken()
+        {
+            ParseArgs();
+            return tdkAuthToken;
+        }
+
+        public static string GetWalletAddressFromJwt()
+        {
+            // TODO check token expiration date?
```

Comment:
Not sure if needed, if we expect the launcher to pass a valid token. Not much we can do if its expired either way right?

Reply:
Could throw an error and have the game developer show some UI that the game needs to be relaunched? Otherwise we'd need a refresh token endpoint to call, right?

Reply:
Is the token generated when the game is launched or when you sign in on the launcher? If the token lasts relatively long it might be fine to throw an error, since it would be unlikely

Reply:
It's created by the launcher when you sign-in, not on launch